### PR TITLE
modify setup and docs to use new repo

### PR DESCRIPTION
**Pull request by mchestnut91**
This PR updates the setup.py file and the docs to reference the new repo. I believe these changes are sufficient for us to be able to use pypi with this repo, but we will need to test once we've done a new release

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -26,8 +26,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'btrdb'
-copyright = '2019, Michael P. Andersen'
-author = 'Michael P. Andersen'
+copyright = '2021, PingThingsIO'
```
Comment (looselycoupled) [inline docs/source/conf.py:29]: pingthings here should be referred to as `Ping Things, Inc.`.  Note the space.

**Comment by looselycoupled** [comment]
You should also look at the following file for changes: https://github.com/PingThingsIO/btrdb-python/blob/master/docs/source/conf.py

**Comment by looselycoupled** [comment]
Also, perhaps a new ticket but the URL for the image/badge concerning build status is still pointing to travis and uses the wrong github repo.  https://raw.githubusercontent.com/PingThingsIO/btrdb-python/master/docs/source/index.rst

File: docs/source/conf.py (hunk 1 above)
Comment (mchestnut91) [reply docs/source/conf.py:29]: sounds good. Should I change it to that everywhere or just the copyright line?

**Comment by mchestnut91** [comment]
FYI I created a new ticket for the build status badge: https://app.clubhouse.io/pingthings-ws/story/13382/update-the-badge-for-build-status-in-btrdb-python-docs

**Review by looselycoupled — APPROVED** [review]
LGTM

**Pull request merged**